src/charm_state.py

```diff
@@ -213,9 +219,17 @@ def from_charm(
             )
             error_field_str = " ".join(f"flask_{f}" for f in error_fields)
             raise CharmConfigInvalidError(f"invalid configuration: {error_field_str}") from exc
+        database_migration_script = charm.config.get("database_migration_script")
+        if database_migration_script:
+            database_migration_script = os.path.normpath(FLASK_APP_DIR / database_migration_script)
+            if not database_migration_script.startswith(str(FLASK_APP_DIR)):
+                raise CharmConfigInvalidError(
+                    "database_migration_script is not inside /srv/flask/app"
```

Comment:
Should the `/srv/flask/app` be hardcoded?

Reply:
Yes, indeed, updated, thanks!